install.sh

```diff
@@ -226,6 +243,7 @@ nvm_do_install() {
     fi
   fi
 
+  nvm_bash_completion
```

Comment:
hmmmm ... I don't think that's **guess**, existing rc files mean that the user will use that shell.